helpers/calendar/events.py

```diff
@@ -90,3 +90,50 @@ def get_all_room_schedules(self, query, start_date, end_date):
                 all_events.append(current_event)
 
         return all_events, all_dates
+
+    def get_room_index(self, attendees, calendar_id):
+        index_of_room = 0
+        for index in range(len(attendees)):
+            if attendees[index]["email"] == calendar_id:
+                index_of_room = index
+                break
+        return index_of_room
+
+    def get_all_recurring_events(self, query, start_date, end_date):
+        """ get all recurring events"""
+        rooms = query.filter_by(state="active")
+        credentials = Credentials()
+        service = credentials.set_api_credentials()
+        recurring_events = []
+        for room in rooms:
+            events = []
+            try:
+                events = service.events().list(
+                    calendarId=room.calendar_id, timeMax=end_date,
+                    timeMin=start_date, singleEvents=True,
+                    orderBy='startTime').execute()
+            except Exception:
+                continue
+            for event in events['items']:
+                check_recurring_event = event.get("recurringEventId")
```

Comment:
Are we checking recurring events or getting them then assigning them to this variable? If the latter then the variable name shouldn't be check_recurring_event as this refers to an action and not a noun. Use descriptive variable names.

Reply:
Noted...Implementing it asap